packages/pancake-profile-sdk/src/utils/addressHelpers.ts

```diff
@@ -0,0 +1,16 @@
+import addresses from "../constants/contracts";
+import { Address } from "../types";
+
+export const getAddress = (address: Address): string => {
+  const mainNetChainId = 56;
+  const chainId = process.env.REACT_APP_CHAIN_ID;
```

Comment:
Reminder, we won't be using `REACT_APP` prefix for this (right?).

Reply:
@hachiojidev yeah it could be set via constructor as I mentioned in slack.
Something like
```js
import PCSProfile from "@pancakeswap-libs/profile-sdk"
const httpProvider = new Web3.providers.HttpProvider("http://customrpc.com, {
  timeout: 10000,
});
const myWeb3 = new Web3(httpProvider);
const chainId = 97
const profile = new Profile({ web3: myWeb3, chainId })
const username = profile.getProfile("0x1234")
```

What do you think?